packages/utils/src/estimate-gas-fees.ts

```diff
@@ -15,23 +15,20 @@ import { LogTypes } from '@requestnetwork/types';
  */
 async function estimateGasFees({
```

Comment:
Suggestion: I would rename this method from `estimateGasFees` to `normalizeGasFees`, since it doesn't have the responsibility of gas estimation anymore, what do you think?
_PS: the JSDoc is out of date_